WordPress/wordpress-playground
# OPFS: Migrate to the latest spec

## Description

Previous implementation of OPFS sync used deprecated webkitRequestFileSystem and only worked in Chrome-based browser.

This commit migrates to the modern FileSystem API which means it can be also used on Safari, Firefox, and mobile:

https://developer.mozilla.org/en-US/docs/Web/API/File_System_Access_API

<img width="1135" alt="CleanShot 2023-06-14 at 16 53 50@2x" src="https://github.com/WordPress/wordpress-playground/assets/205419/1c1a6369-d5a0-47c3-971c-a471feaf7ff0">


## Testing Instructions

Select "Persistent site" in the mode picker and confirm your changes are preserved between page refreshes:

1. Create a post
2. Install a plugin
3. Delete a plugin

cc @dmsnell
